### ISSUE TITLE
docker-compose, there is no template

### ISSUE THREAD

**Issue by csakaszamok** [coder/coder#5472]
When you start a coder server with docker-compose you don't have template by default, and you can't create one.
The gui says just run `coder templates init` but you cannot execute this command in container.
(say the true I tried, but it demands extra steps like `cd ./docker-image-builds`, login, etc.)

How can I add templates to the coder server at first time (in a docker-compose environment)

**Comment by bpmct** [comment]
> How can I add templates to the coder server at first time (in a docker-compose environment)

We're adding a web UI to add templates, so that will be in soon! In the meantime, you will need to download the CLI on your local machine and log into the server from the CLI

```sh
curl -L https://coder.com/install.sh | sh
coder login <URL>
coder templates init
coder templates create
```

Going to close this for now, but think about ways we can make the instructions clear! This is something we've heard a few times.

**Comment by matifali** [comment]
This has been made available in version 0.13.6
